src/styles/index.scss

```diff
@@ -1,3 +1,69 @@
-body {
+@import './variable';
```

Comment:
Ensure that the `./variable` file exists and contains all necessary variable definitions. Missing variables will cause the SCSS to fail during compilation.